File: netlify/functions/watchlist.js
const fs = require('fs');
const path = require('path');

exports.handler = async () => {
  const file = path.join(__dirname, '../../backend/watchlist.json');
  const items = JSON.parse(fs.readFileSync(file, 'utf8'));
  return {
    statusCode: 200,
    headers: {'Content-Type': 'application/json; charset=utf-8', 'Cache-Control': 'no-store', 'Access-Control-Allow-Origin': '*'},
    body: JSON.stringify({items})
  };
};

File: netlify/functions/watchlist.py
import json
from pathlib import Path

ROOT = Path(__file__).resolve().parents[2]

def handler(event, context):
    data = json.loads((ROOT / 'backend' / 'watchlist.json').read_text(encoding='utf-8'))
    return {
        'statusCode': 200,
        'headers': {'Content-Type': 'application/json; charset=utf-8', 'Cache-Control': 'no-store', 'Access-Control-Allow-Origin': '*'},
        'body': json.dumps({'items': data}, ensure_ascii=False)
    }
